README.md

```diff
@@ -566,7 +592,21 @@ Forked from [AirBNB's style guide](https://github.com/airbnb/javascript)
   count();  // 3
   ```
 
-- [7.9](#7.9) <a name='7.9'></a> Never use the Function constructor to create a new function.
+  - [7.9](#7.9) <a name='7.9'></a> Always put default parameters last.
+
+    ```javascript
+    // bad
+    function handleThings(opts = {}, name) {
+      // ...
+    }
+
+    // good
+    function handleThings(name, opts = {}) {
+      // ...
+    }
+    ```
```

Comment:
I'm not so sure about this one. Sometimes consistency can make more sense. For example, say I have an external API that just calls an internal one, and the internal one defaults things. 

ie:
```js
@public
function foo(bar, baz) {
  internalFoo(bar, baz);
}

@private
function internalFoo(bar = 1, baz) {
  // things
}
```

In situations like these, I don't want to flip the signatures, because then I would just get confused about which one goes where all the time. We have implementations like this in Video Interface. So I don't think it makes much sense, nor is really necassary.